src/client/trackers/real-user-performance/documentReady.js

```diff
@@ -0,0 +1,15 @@
+export const isReady = () => document.readyState === 'complete';
```

Comment:
i think fine to keep this until we decide about https://github.com/Financial-Times/dotcom-page-kit/issues/740